airbyte-commons-worker/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -501,11 +502,9 @@ public KubePodProcess(final boolean isOrchestrator,
         .withLabels(labels)
         .withAnnotations(annotations)
         .endMetadata()
-        .withNewSpec();
-
-    if (isOrchestrator) {
-      podBuilder = podBuilder.withServiceAccount("airbyte-admin").withAutomountServiceAccountToken(true);
-    }
+        .withNewSpec()
+        .withServiceAccount(isOrchestrator ? "airbyte-admin" : serviceAccount)
```

Comment:
Should we add a dedicated variable for the orchestrator SA and well as the one added in order to have something like:
`.withServiceAccount(isOrchestrator ? orchestratorServiceAccount : serviceAccount)`. If we go that way, we need to make sure that the default value is `"airbyte-admin"`.